bigtable/client/filters.h

```diff
@@ -71,8 +74,12 @@ class Filter {
     return tmp;
   }
 
-  /// Create a filter that accepts only the last @a n values.
-  static Filter Latest(int n) {
+  /**
```

Comment:
Do we have a pattern for when we use `/** ... */` vs. when we use `///` comments? Does Doxygen handle both of them? Do they behave differently somehow?

This should be documented in the style guide, and we should be consistent, assuming they are identical.

Reply:
The style guide proposed in #98 includes guidelines for when to use `/** */` vs. `///` and yes, Doxygen supports both.